web/template/add-user.gotmpl

```diff
@@ -0,0 +1,167 @@
+{{ template "page" . }}
+
+{{ define "backlink" }}
+  <a class="govuk-back-link" href="{{ prefix "/users" }}">Back</a>
+{{ end }}
+
+{{ define "title" }}{{ if .Errors }}Error: {{ end }}Add user{{ end }}
+
+{{ define "main" }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ template "error-summary" .Errors }}
+
+      <h1 class="govuk-heading-xl">Add new user</h1>
+      
+      <form class="form" action="{{ prefix "/add-user" }}" method="post">
+        <div class="govuk-form-group {{ if .Errors.email }}govuk-form-group--error{{ end }}">
+          <label class="govuk-label" for="f-email">Email address</label>
+          {{ range .Errors.email }}
+            <span class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span> {{ . }}
+            </span>
+          {{ end }}
+          <input class="govuk-input" id="f-email" name="email" type="text">
```

Comment:
Just realised that this (and others) technically needs the `govuk-input--error` class if there's an error:
```suggestion
          <input class="govuk-input {{ if .Errors.email }}govuk-input--error{{ end }}" id="f-email" name="email" type="text">
```

Could cycle back round on that though